### PR TITLE
feat: product details

### PR DESCRIPTION
**Pull request by sonishreyas**
## Change logs
- Home page added. 
- Since many of my components in the component library are not merged I have kept a copy of them in this repo. Once they are merged I will remove them from this repo.
- Have not worked on the responsiveness of the page will work on this after completing the assignment

## Screenshot
image

## Preview Link
https://deploy-preview-8--ripple-store.netlify.app/pages/product-details/product-details.html

### CODE REVIEW

File: css/lists.css [hunk 1]
```diff
@@ -24,3 +24,35 @@
 .spaced-list li {
 	margin: 1rem;
 }
+
+.dropdown li {
+	width: 1rem !important;
```
Comment (ravikumar0403) [inline css/lists.css:29]: Try not to use important, See if you can achieve the outcome by changing the specificity

File: css/lists.css [hunk 2]
```diff
@@ -24,3 +24,35 @@
...
+	padding: 0;
+	margin: 0;
+}
+.dropdown:hover .dropdown-list {
+	display: flex;
+}
+
+.dropdown:hover {
+	display: flex !important;
+}
```
Comment (ravikumar0403) [inline css/lists.css:58]: here as well. I guess increasing the specificity can do the job

**Review by ravikumar0403 — COMMENTED** [review]
Looks good. Check for the `!important` rule though

**Review by rishmi5h — COMMENTED** [review]
I guess you forgot to add cursor: pointer; in select size and wishlist buttons.
and for add to cart, it's there only when we hover above the cart icon. 
Look into it rest looks good.

**Comment by sonishreyas** [comment]
> I guess you forgot to add cursor: pointer; in select size and wishlist buttons. and for add to cart, it's there only when we hover above the cart icon. Look into it rest looks good.

Thanks will look into this

File: css/lists.css (hunk 2 above)
Comment (sonishreyas) [reply css/lists.css:58]: ok will change that

**Pull request merged**